src/app.ts

```diff
@@ -18,17 +20,36 @@ const main = async () => {
 
   await dbConnection();
 
+  var bodyParser = require('body-parser');
+
+  var app = Express();
+
+  app.use(bodyParser.urlencoded({ extended: false }))
+  app.use(bodyParser.json())
+
   const server = new ApolloServer({
     schema,
     playground: true,
     introspection: true,
   });
-  const app = Express();
 
   app.get('/', (req, res) => {
     res.sendFile('index.html', { root: __dirname });
   });
 
+  // Setup Firebase Admin
+  if (process.env.NODE_ENV === 'production') {
+    admin.initializeApp({
+      credential: admin.credential.cert(process.env.FCM_AUTH_KEY_PATH),
+    });
+  }
+
+  app.post('/collect', (req, res) => {
```

Comment:
shouldn't there be a trailing slash here? `/collect/`?

Reply:
<img width="544" alt="image" src="https://user-images.githubusercontent.com/94572978/164946803-190f26b2-b485-4f3e-b3a0-c7a3dc99b8ce.png">

no i think the endpoint doesn't have a trailing slash

Reply:
you need to have a trailing slash though, that's appdev standard

Reply:
i think it's also backend standard..